tests/library/chromium/chromium.spec.ts

```diff
@@ -128,6 +129,61 @@ test('should not create a worker from a shared worker', async ({ page, server })
   expect(serviceWorkerCreated).not.toBeTruthy();
 });
 
+test('should emit console messages from service worker', async ({ page, server }) => {
+  const [worker] = await Promise.all([
+    page.context().waitForEvent('serviceworker'),
+    page.goto(server.PREFIX + '/serviceworkers/empty/sw.html')
+  ]);
+
+  const [consoleMessage] = await Promise.all([
+    new Promise<ConsoleMessage>(resolve => worker.once('console', resolve)),
+    worker.evaluate(() => console.log('hello from service worker', {
```

Comment:
Let's also add a test that console messages logged on service worker start are also properly reported.